packages/synapse-react-client/src/components/Header/Header.tsx

```diff
@@ -0,0 +1,87 @@
+import { Box, Stack, Typography } from '@mui/material'
+
+export type HeaderProps = {
+  backgroundImage: string
+  title?: React.ReactNode
+  subTitle?: React.ReactNode
+  description?: React.ReactNode
+}
+
+const Header = ({
```

Comment:
Maybe rename to something like PortalHomePageHeader? Lots of pages/sites will have different headers, so if this is living in SRC it should have a more specific name